tensorflow/core/kernels/depthwise_conv_grad_op.cc

```diff
@@ -611,7 +611,8 @@ class DepthwiseConv2dNativeBackpropInputOp : public OpKernel {
     // If in_depth==1, this operation is just a standard convolution.
     // Depthwise convolution is a special case of cuDNN's grouped convolution.
     bool use_cudnn = use_cudnn_ && (in_depth == 1 || (use_cudnn_grouped_conv_ &&
-        IsCudnnSupportedFilterSize(filter_rows, filter_cols)));
+        IsCudnnSupportedFilterSize(filter_rows, filter_cols, in_depth,
+                                   out_depth)));
```

Comment:
Btw, since all four params are of type `int32`, it is easy to accidentally switch up the order.  What do you think about rewriting these calls to `IsCudnnSupportedFilterSize(/*filter_rows=*/filter_rows, /*filter_cols=*/filter_cols, /*in_depth=*/in_depth, /*out_depth=*/out_depth)`?  I don't know if these are useful in the open source build, but for Google-internal builds we check that these actually match the parameter names.

Reply:
Sure. Done. PTAL.